cmd/sanitize-prow-jobs/main.go

```diff
@@ -6,36 +6,121 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strings"
 	"sync"
 
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	prowconfig "k8s.io/test-infra/prow/config"
 	"sigs.k8s.io/yaml"
 )
 
-const defaultCluster = "api.ci"
-
 type options struct {
 	prowJobConfigDir string
+	configPath       string
 
 	help bool
 }
 
+// Config is the configuration file of this tools, which defines the cluster parameter for each Prow job, i.e., where it runs
+type Config struct {
+	// the cluster context name if no other condition matches
+	Default string
```

Comment:
Did we lose the JSON tags?

Reply:
"sigs.k8s.io/yaml"

This pkg seems working without json tags for those fields.
(tested with config in https://github.com/openshift/release/pull/8209)

Reply:
Sure, there are defaults when your field names are the same, but please follow the convention we use literally everywhere else -- explicitly set them.

Reply:
Done.